shell/platform/tizen/tizen_renderer_egl.cc

```diff
@@ -63,11 +74,27 @@ bool TizenRendererEgl::CreateSurface(void* render_target,
   {
     const EGLint attribs[] = {EGL_NONE};
 
-    auto* egl_window =
-        static_cast<EGLNativeWindowType*>(ecore_wl2_egl_window_native_get(
-            static_cast<Ecore_Wl2_Egl_Window*>(render_target)));
-    egl_surface_ =
-        eglCreateWindowSurface(egl_display_, egl_config_, egl_window, attribs);
+    if (render_target_display) {
+      auto* egl_window =
+          static_cast<EGLNativeWindowType*>(ecore_wl2_egl_window_native_get(
+              static_cast<Ecore_Wl2_Egl_Window*>(render_target)));
+      egl_surface_ = eglCreateWindowSurface(egl_display_, egl_config_,
+                                            egl_window, attribs);
+    } else {
+#ifdef NUI_SUPPORT
+      Dali::NativeImageSourceQueuePtr dali_native_image_queue_ =
+          static_cast<Dali::NativeImageSourceQueue*>(render_target);
+      tbm_surface_queue_h tbm_surface_queue_ =
```

Comment:
Remove the trailing underscores in the names.

Reply:
done